### PR TITLE
Dockerfile arm

### PR DESCRIPTION
**Pull request by cedricfarinazzo**
I added a dockerfile for arm proccesors.

#25

### CODE REVIEW

**Comment by GerardSmit** [comment]
Please target the `dev` branch. Also it seems you've used the Dockerfile from the `master` branch, which is outdated.
Check https://github.com/AeonLucid/Impostor/blob/dev/Dockerfile for the new Dockerfile.

**Comment by cedricfarinazzo** [comment]
I have update the dockerfile. It should be good now.
Do I have to update github actions config?

File: Dockerfile.arm [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    dotnet restore -r linux-arm ./src/Impostor.Shared/Impostor.Shared.csproj && \
+    dotnet restore -r linux-arm ./submodules/Hazel-Networking/Hazel/Hazel.csproj
+
+# Copy everything else.
+COPY submodules/. ./submodules/
+COPY src/. ./src/
+RUN dotnet publish -c release -o /app -r linux-arm --no-restore ./src/Impostor.Server/Impostor.Server.csproj
+
+# Final image.
+FROM mcr.microsoft.com/dotnet/core/runtime-deps:latest
```
Comment (GerardSmit) [inline Dockerfile.arm:19]:
```suggestion
FROM mcr.microsoft.com/dotnet/core/runtime-deps:3.1-alpine
```

We don't really want to use `latest` here incase a breaking change gets be released in `runtime-deps`.

**Comment by GerardSmit** [comment]
There is also a merging conflict because the source branche is from `master`.
If you pull the `dev` branch to your branch, you can merge it.

File: Dockerfile.arm (hunk 1 above)
Comment (cedricfarinazzo) [reply Dockerfile.arm:19]: We cannot use the alpine version. I don't know if linux-arm runtime identifier works with alpine image. Maybe using 3.1 is a better solution.
```suggestion
FROM mcr.microsoft.com/dotnet/core/runtime-deps:3.1
```

**Pull request closed without merging**